distributed/scheduler.py

```diff
@@ -5976,10 +6009,7 @@ def _reevaluate_occupancy_worker(self, ws: WorkerState):
         new = 0
         nbytes = 0
         for ts in ws._processing:
-            duration = self.get_task_duration(ts)
-            comm = self.get_comm_cost(ts, ws)
-            ws._processing[ts] = duration + comm
```

Comment:
Do we still want to set this?

Reply:
Yes we still want to set `ws._processing`, but it's been moved into a `set_duration_estimate` method:

https://github.com/dask/distributed/blob/308c4df69e250d90a159885fd566d253228b48a7/distributed/scheduler.py#L4714-L4729